src/icons/fileIcons.ts

```diff
@@ -470,7 +472,8 @@ export const fileIcons: FileIcons = {
         'mrf',
       ],
     },
-    { name: 'lib', fileExtensions: ['lib', 'bib'] },
+    { name: 'lib', fileExtensions: ['lib', 'bib', 'a'] },
```

Comment:
the a file extension is too generic I think. Is it really only related to lib files?

Reply:
You would think so, but surprisingly no! The overwhelming usecase for the `.a` extension is Unix static libraries. It's in the same camp as the `.o` extension, which is the Unix compiled object equivalent, and that was added without issue